scripts/swupd-performance/swupdservicesetup.sh

```diff
@@ -0,0 +1,74 @@
+#!/usr/bin/bash
+
+set -e
+
+echo "Setup staging directory"
+
+if [ -d "$HOME/staging" ]; then
+	echo "Staging directory already exists" || true
+else
+	dir=$HOME/staging
+	mkdir $HOME/staging
+
+	if [ $? -eq 0 ]; then
+		echo "Staging directory: $dir successfully created!"
+	fi
+fi
+
+echo ""
+echo "Setup performance monitoring script config dir"
+
+if [ -d /etc/swupd_perform_update_config ]; then
+	echo "Config directory already exists" || true
+else
+	mkdir /etc/swupd_perform_update_config
+fi
+
+cp -r stagingfilegen.sh /etc/swupd_perform_update_config/
+
+# Setting the systemd for the swupd-update-test.service
+cat <<EOF > /etc/systemd/system/swupd-update-test.service
+[Unit]
+Description=Update Software content
+
+[Service]
+Type=oneshot
+ExecStartPre=/etc/swupd_perform_update_config/stagingfilegen.sh
+EnvironmentFile=/etc/swupd_perform_update_config/script_config
+ExecStart=/usr/bin/swupd update --json-output --time
```

Comment:
I prefer to just set this to

    ExecStart=/usr/bin/bash -c 'usr/bin/swupd info > /directory/swupd_output_$(/usr/bin/date \+%%Y%%m%%d_%%H%%M%%S)'

Then we don't need an extra script to run date and save that to a var

Reply:
I can certainly try it...but I think systemd might have problem with it.

Reply:
Actually , I see a problem with that. in case we change the directory, it requires a change to file and a daemon-reload. However if I read only Environment variable, then we chnage it in that file keep the system unit constant.  I think we should keep the systemd unit file clean

Reply:
It makes sense. Let's go with your solution then.